components/proxy/src/main/java/com/hotels/styx/admin/AdminServerBuilder.java

```diff
@@ -123,6 +124,7 @@ public HttpServer build() {
         httpRouter.add("/admin/origins/status", new OriginsInventoryHandler(environment.eventBus()));
         httpRouter.add("/admin/configuration/logging", new LoggingConfigurationHandler(styxConfig.startupConfig().logConfigLocation()));
         httpRouter.add("/admin/configuration/startup", new StartupConfigHandler(styxConfig.startupConfig()));
+        httpRouter.add("/admin/proxystatus", new ProxyStatusHandler(environment.eventNexus()));
```

Comment:
I would consider adding more structure in the admin interface endpoint names. I would call this "/admin/proxy/server/status" etc.

Where:
- `/proxy` refers to the styx proxy application,
- `/proxy/server` refers to the `server` attribute inside proxy application,
- `/proxy/server/status` refers to the status attribute of the proxy server.

and so on.

@dvlato what do you think? I believe you are the most experienced amongst us in the web API design?

Reply:
It sounds fine for me. I don't know if the status belongs only to the 'server attribute'  or we can consider it as the whole status for the whole 'proxy (server)'. I mean, would we have proxy/<something_different_from_server>/status ? And, in that case, do we need to aggregate the status to find out what's the global status of the proxy?